x-pack/test/functional/page_objects/security_page.ts

```diff
@@ -17,53 +18,114 @@ export function SecurityPageProvider({ getService, getPageObjects }) {
   const userMenu = getService('userMenu');
   const PageObjects = getPageObjects(['common', 'header', 'settings', 'home', 'error']);
 
-  class LoginPage {
-    async login(username, password, options = {}) {
-      const [superUsername, superPassword] = config.get('servers.elasticsearch.auth').split(':');
+  interface LoginOptions {
```

Comment:
__note:__ I wanted to replace it with `LoginExpectedResult` initially, but it's used in too many places right now.